s3/public.go

```diff
@@ -298,6 +317,17 @@ func (storage *PublishedStorage) LinkFromPool(publishedDirectory, fileName strin
 	sourceMD5 := sourceChecksums.MD5
 
 	if exists {
+        if len(destinationMD5) != 32 {
```

Comment:
This seems wrongly indented. Please check with "go fmt".

Reply:
Ack, updating,